src/components/playback/DrawManager.js

```diff
@@ -55,6 +55,8 @@ export default class DrawManager {
   }
 
   setPosition (position) {
+    position = clamp(position, 0, this.duration)
```

Comment:
This may seem like it doesn't belong, but it's actually related to some resizing of the progress bars I did on the stream page.